tasks/Module_2/task_1.py

```diff
@@ -0,0 +1,47 @@
+"""Task 1 and 2 in module 2."""
+
+"""Constants and filters/"""
```

Comment:
This is more of a code comment, it should be created like
`# Constants and filters`